### ISSUE TITLE
Fontsize problem in <rend> since 3.13

### ISSUE THREAD

**Issue by wergo** [rism-digital/verovio#3140]
Having mutliple `<rend>` elements with different fontsize attributes are not rendered correctly in Verovio 3.13. This example has nested rend elements with mutliple relative fontsize attributes in a `<pgHead>` and in a `<dir>`:  

<img width="721" alt="Bildschirmfoto 2022-11-25 um 10 38 35" src="https://user-images.githubusercontent.com/45632600/203948861-9db01e46-e9b9-4c63-a692-11c244f9b5ca.png">

Correct/expected behavior is observed in Verovio up to version 3.12.1: 

<img width="722" alt="Bildschirmfoto 2022-11-25 um 10 40 31" src="https://user-images.githubusercontent.com/45632600/203949167-08ca1195-bbbd-4440-a7e0-1988f930c99a.png">

<details>
<summary>Example MEI: </summary>

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/dev/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/dev/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="5.0.0-dev">
   <meiHead>
      <fileDesc>
         <titleStmt>
            <title>Nexting rend and font size</title>
         </titleStmt>
         <pubStmt>
            <respStmt>
               <persName role="encoder">Laurent Pugin</persName>
            </respStmt>
            <date isodate="2022-10-06" />
         </pubStmt>
         <seriesStmt>
            <title>Verovio test suite</title>
         </seriesStmt>
         <notesStmt>
            <annot>
               <p>Verovio font size applies to the children of a rend too.</p>
            </annot>
         </notesStmt>
      </fileDesc>
      <encodingDesc>
         <appInfo>
            <application version="3.7.0" label="2">
               <name>Verovio</name>
            </application>
         </appInfo>
      </encodingDesc>
   </meiHead>
   <music>
      <body>
         <mdiv xml:id="mn8v3bo">
            <score xml:id="s36zjwv">
               <scoreDef xml:id="sy0ogwt">
                  <pgHead>
                     <rend valign="middle" halign="center" fontsize="100">Title at 100%<lb /><rend fontsize="150%"> at 150%</rend><lb /><rend fontsize="150%"> at 150%</rend><lb ></lb><rend fontsize="150%"> at 150%</rend></rend>
                  </pgHead>
                  <staffGrp xml:id="sjkroe1">
                     <staffDef xml:id="swfg6qs" n="1" lines="5">
                        <clef xml:id="c162i4bp" shape="G" line="2" />
                     </staffDef>
                  </staffGrp>
               </scoreDef>
               <section xml:id="s1824cwy">
                  <pb xml:id="popcg5v" />
                  <measure xml:id="m1ueby01">
                     <staff xml:id="sqgac2x" n="1">
                        <layer xml:id="l3xb2zy" n="1">
                           <note xml:id="n18ktvhq" dur="4" oct="4" pname="e" />
                           <note xml:id="na38spz" dur="4" oct="4" pname="e" />
                           <note xml:id="n9mrg49" dur="4" oct="4" pname="e" />
                           <note xml:id="n10kfyjy" dur="4" oct="4" pname="e" />
                           <note xml:id="niasouc" dur="4" oct="4" pname="e" />
                        </layer>
                     </staff>
                     <dir xml:id="dklevtu" place="above" staff="1" tstamp="1.000000">
                        <rend xml:id="r1jxial2" rend="box" fontsize="50.00%">Text at 50% with <rend xml:id="rdmhcvm" fontsize="200.00%">text at 200%</rend><rend xml:id="rdmhcvm" fontsize="150.00%">text at 150%</rend><rend xml:id="rdmhcvm" fontsize="125.00%">text at 125%</rend> but not all</rend>
                     </dir>
                     <dir xml:id="dc3ella" place="below" staff="1" tstamp="1.000000">
                        <rend xml:id="r1mneli8" fontsize="75.00%">Text at 75% with <rend xml:id="r14pbuvn" fontsize="smaller">smaller <rend xml:id="r19am4bz" fontsize="smaller">and even smaller</rend>
                           </rend> text</rend>
                     </dir>
                  </measure>
               </section>
            </score>
         </mdiv>
      </body>
   </music>
</mei>
```
</details>

It seems that fontsize is taken relative to previous rend element, instead of relative to an absolute 100% font size. 

(Additionally, line spacing seems a bit condensed and should be addressed in another issue.)

**Comment by lpugin** [comment]
As a first guess it is related to [this](https://github.com/rism-digital/verovio/pull/3058) change.

Line spacing should definitely be addressed. Having the font size relative to the previous element looks right to me, though.

**Comment by lpugin** [comment]
Also, it works properly if the `<lb>` is put within the `<rend>`. 

![image](https://user-images.githubusercontent.com/689412/203961626-a3f521e0-9aad-4edb-b559-baffff3eb4a4.png)

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="https://music-encoding.org/schema/dev/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="https://music-encoding.org/schema/dev/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="5.0.0-dev">
   <meiHead>
      <fileDesc>
         <titleStmt>
            <title>Nexting rend and font size</title>
         </titleStmt>
         <pubStmt>
            <respStmt>
               <persName role="encoder">Laurent Pugin</persName>
            </respStmt>
            <date isodate="2022-10-06" />
         </pubStmt>
         <seriesStmt>
            <title>Verovio test suite</title>
         </seriesStmt>
         <notesStmt>
            <annot>
               <p>Verovio font size applies to the children of a rend too.</p>
            </annot>
         </notesStmt>
      </fileDesc>
      <encodingDesc>
         <appInfo>
            <application version="3.7.0" label="2">
               <name>Verovio</name>
            </application>
         </appInfo>
      </encodingDesc>
   </meiHead>
   <music>
      <body>
         <mdiv xml:id="mn8v3bo">
            <score xml:id="s36zjwv">
               <scoreDef xml:id="sy0ogwt">
                  <pgHead>
                     <rend valign="middle" halign="center" fontsize="100%">Title at 100%<rend fontsize="150%"><lb /> at 150%</rend><rend fontsize="150%"><lb /> at 150%</rend><rend fontsize="150%"><lb /> at 150%</rend></rend>
                  </pgHead>
                  <staffGrp xml:id="sjkroe1">
                     <staffDef xml:id="swfg6qs" n="1" lines="5">
                        <clef xml:id="c162i4bp" shape="G" line="2" />
                     </staffDef>
                  </staffGrp>
               </scoreDef>
               <section xml:id="s1824cwy">
                  <pb xml:id="popcg5v" />
                  <measure xml:id="m1ueby01">
                     <staff xml:id="sqgac2x" n="1">
                        <layer xml:id="l3xb2zy" n="1">
                           <note xml:id="n18ktvhq" dur="4" oct="4" pname="e" />
                           <note xml:id="na38spz" dur="4" oct="4" pname="e" />
                           <note xml:id="n9mrg49" dur="4" oct="4" pname="e" />
                           <note xml:id="n10kfyjy" dur="4" oct="4" pname="e" />
                           <note xml:id="niasouc" dur="4" oct="4" pname="e" />
                        </layer>
                     </staff>
                     <dir xml:id="dklevtu" place="above" staff="1" tstamp="1.000000">
                        <rend xml:id="r1jxial2" rend="box" fontsize="50.00%">Text at 50% with <rend xml:id="rdmhcvm" fontsize="200.00%">text at 200%</rend><rend xml:id="rdmhcvm" fontsize="150.00%">text at 150%</rend><rend xml:id="rdmhcvm" fontsize="125.00%">text at 125%</rend> but not all</rend>
                     </dir>
                     <dir xml:id="dc3ella" place="below" staff="1" tstamp="1.000000">
                        <rend xml:id="r1mneli8" fontsize="75.00%">Text at 75% with <rend xml:id="r14pbuvn" fontsize="smaller">smaller <rend xml:id="r19am4bz" fontsize="smaller">and even smaller</rend>
                        </rend> text</rend>
                     </dir>
                  </measure>
               </section>
            </score>
         </mdiv>
      </body>
   </music>
</mei>
```

**Comment by lpugin** [comment]
The behaviour is similar to CSS rendering
```css
div {
  font-size: 150%;
}
```
```html
<div>
  Text
  <div>
    Nested text
    <div>
      More nested text
    </div>
  </div>
</div>
```
![image](https://user-images.githubusercontent.com/689412/203964104-e1a81296-6dac-4eca-bd85-36bc43aa711a.png)

**Comment by wergo** [comment]
Ah, I see; thanks for this update and for the `<lb>` clarification.

But my example above is only nested in one layer; the other `<rend>` elements are in parallel.

The html analogy would be: 

```html 
<div>
    Text
    <div>Nested text</div>
    <div>Nested text</div>
    <div>Nested text</div>
</div>
```
and Firefox renders it, how I would expect it to render:

<img width="195" alt="Bildschirmfoto 2022-11-25 um 12 18 08" src="https://user-images.githubusercontent.com/45632600/203974486-90aecfa9-93b6-46b6-9be9-5383ffc63461.png">

**Comment by wergo** [comment]
That works great now! Thanks a lot!
